CMake/SlicerMacroBuildApplication.cmake

```diff
@@ -540,7 +540,16 @@ macro(slicerMacroBuildApplication)
       # Define list of extra 'application to launch' to associate with the launcher
       # within the build tree
       set(extraApplicationToLaunchListForBuildTree)
-      if(EXISTS "${QT_DESIGNER_EXECUTABLE}")
+
+      if(${Slicer_REQUIRED_QT_VERSION} VERSION_GREATER_EQUAL 5)
```

Comment:
Maybe you could check for QT_DESIGNER_EXECUTABLE variable and if it is defined and non-empty then just use that?